src/generate_test_vectors.py

```diff
@@ -1174,7 +1174,7 @@ def formatResults(kem, caSK, ct, ss ):
 
   # for standalone ML-KEM, we need to wrap the private key in an OCTET STRING, but not when it's a composite
```

Comment:
Per my other comment on this review, I believe that `dk` should just be the raw bytes.  Hence the standalone KEM logic on line 1159 should be removed and the raw bytes always set for `jsonTest['dk']`.

Only when encoding the key into PKCS#8 does there need to be different logic.  Here on line 1175 the comment should be changed from

> \# for standalone ML-KEM, we need to wrap the private key in an OCTET STRING, but not when it's a composite

to

> \# for standalone ML-KEM, we need to encode the private key using ML-DSA's CHOICE encoding, but not when it's a composite